move/abi/sources/abi.move

```diff
@@ -358,3 +351,65 @@ fun test_multiple() {
     assert!(reader.read_vector_u256() == input3);
     assert!(reader.read_vector_bytes() == input4);
 }
+
+#[test]
+fun test_raw_struct() {
+    let number = 3;
+    let data = b"data";
+
+    let mut writer = new_writer(1);
+    let mut struct_writer = new_writer(2);
+    struct_writer.write_u256(number).write_bytes(data);
+    writer.write_bytes_raw(struct_writer.into_bytes());
+
+    let bytes = writer.into_bytes();
+
+    let mut reader = new_reader(bytes);
+
+    let struct_bytes = reader.read_bytes_raw();
+
+    let mut struct_reader = new_reader(struct_bytes);
```

Comment:
`write_bytes_raw(..)` -> `new_writer_wrapper(struct_writer)`
something like this? `new_writer_from_writer`, `new_writer_nested`

Reply:
not always the case unfortunately: for structs another writer would be desirable, since the raw bytes are just some abi encoded bytes. For vectors on the other hand the raw bytes consist of the length, followed by some encoded bytes (where the encoding starts matters for offsets).